### ISSUE TITLE
Keyboard focus not indicated in UI

### ISSUE THREAD

**Issue by RocketMan** [RocketMan/zookeeper#320]
The keyboard focus is not visible in the UI; for example, tabbing to the 'Add' buttons in the playlist editor does not present a visual indication that they are focused.

Expected:  All focusable/tabbable elements should show visible indication of focus.
